app/models/booking.rb

```diff
@@ -0,0 +1,7 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :lesson
+  belongs_to :review
+  belongs_to :payment
+  belongs_to :report
```

Comment:
And `optional: true` would be needed here, since by default active record adds a sort of `validates :column, presence: true` to every `belongs_to` relation.

Reply:
@DeFiDegen